tests/pipeline/libvirtd.conf

```diff
@@ -0,0 +1,6 @@
+listen_addr = "127.0.0.1"
```

Comment:
is this part needed at all?

Reply:
Same thoughts to what I said here: https://github.com/karmab/kcli/pull/196#discussion_r418469756